lib/models/factories.js

```diff
@@ -47,7 +47,7 @@ function clientFactory(overrides = {}) {
 
 async function buttonDBFactory(db, overrides = {}) {
   const device = await db.createDevice(
-    DEVICE_TYPE.DEVICE_BUTTON,
```

Comment:
i think its safe to assume that for buttons, you don't have to do an override. think about keeping it at DEVICE_BUTTON as default. should probably ask if this will be too confusing for button and sensor to have different arguments as override.

Reply:
changed in latest push